src/bt_auto_dumper/_v2/__main__.py

```diff
@@ -13,24 +15,33 @@ def main(apiver: str | None = None):
     apiver = apiver or pathlib.Path(__file__).parent.name
     parser = argparse.ArgumentParser(description=f"BT Auto Dumper CLI {apiver}")
     parser.add_argument("--note", help="Comment or note for the operation", type=str, default="")
-    parser.add_argument("--walletname", help="Wallet Name", type=str, required=True)
-    parser.add_argument("--wallethotkey", help="Wallet Hotkey", type=str, required=True)
     parser.add_argument("subnet_identifier", help="Subnet Identifier", type=str)
     parser.add_argument("autovalidator_address", help="AutoValidator Address", type=str)
 
     args = parser.parse_args()
 
-    dump_and_upload(args.walletname, args.wallethotkey, args.subnet_identifier, args.autovalidator_address, args.note)
+    dump_and_upload(args.subnet_identifier, args.autovalidator_address, args.note)
 
 
-def dump_and_upload(walletname, wallethotkey, subnet_identifier: str, autovalidator_address: str, note: str):
+def dump_and_upload(subnet_identifier: str, autovalidator_address: str, note: str):
+    """
+    Dump and upload the output of the commands to the AutoValidator
+    Args:
+        subnet_identifier: Subnet Identifier
+        autovalidator_address: AutoValidator Address
+        note: Comment or note for the operation
+    Example:
+        dump_and_upload("computehorde", "http://localhost:8000", "Test")
+    """
     subnets = {
-        "compute_horde": ["echo 'Mainnet Command 1'", "echo 'Mainnet Command 2'"],
+        "computehorde": ["echo 'Mainnet Command 1'", "echo 'Mainnet Command 2'"],
         "omron": ["echo 'Mainnet Command 1'", "echo 'Mainnet Command 2'"],
     }
-    wallet = bt.wallet(name=walletname, hotkey=wallethotkey)
-    if subnet_identifier in subnets:
-        commands = {subnet_identifier: subnets[subnet_identifier]}
+
+    wallet = bt.wallet(name="validator", hotkey="validator-hotkey")
+    normalized_subnet_identifier = re.sub(r"[_\-.]", "", str.lower(subnet_identifier))
+    if normalized_subnet_identifier in subnets:
+        commands = {normalized_subnet_identifier: subnets[normalized_subnet_identifier]}
```

Comment:
this should not be a dict structure but a list, what's the dict for here? Fix in subsequent PR